On jammy, the compiler (g++-12) does not seem to like fmt's "{:?}"
formatting directives. These yield roundabout errors like
  
  /usr/include/fmt/core.h:2201:22: error: 'constexpr void fmt::v8::basic_format_parse_context<Char, ErrorHandler>::on_error(const char*) [with Char = char; ErrorHandler = fmt::v8::detail::error_handler]' called in a constant expression
   2201 |     context_.on_error(message);
        |     ~~~~~~~~~~~~~~~~~^~~~~~~~~
  /usr/include/fmt/core.h:692:22: note: 'constexpr void fmt::v8::basic_format_parse_context<Char, ErrorHandler>::on_error(const char*) [with Char = char; ErrorHandler = fmt::v8::detail::error_handler]' is not usable as a 'constexpr' function because:
    692 |   FMT_CONSTEXPR void on_error(const char* message) {
        |                      ^~~~~~~~
  /usr/include/fmt/core.h:693:27: error: call to non-'constexpr' function 'void fmt::v8::detail::error_handler::on_error(const char*)'
    693 |     ErrorHandler::on_error(message);
        |     ~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~
  /usr/include/fmt/core.h:618:29: note: 'void fmt::v8::detail::error_handler::on_error(const char*)' declared here
    618 |   FMT_NORETURN FMT_API void on_error(const char* message);
        |                             ^~~~~~~~
  ../../../source/graphics/MapGenerator.cpp:401:110: error: no matching function for call to 'std::runtime_error::runtime_error(<brace-enclosed initializer list>)'
    401 |                 throw std::runtime_error{fmt::format("Loading module {:?} takes too long.", script.string8())};
        |                                                                                                              ^

--- a/source/graphics/MapGenerator.cpp
+++ b/source/graphics/MapGenerator.cpp
@@ -398,7 +398,7 @@ Script::StructuredClone RunMapGeneration
 	scriptInterface.GetContext().RunJobs();
 	auto& future = *compileResult.begin();
 	if (!future.IsDone())
-		throw std::runtime_error{fmt::format("Loading module {:?} takes too long.", script.string8())};
+		throw std::runtime_error{fmt::format("Loading module {} takes too long.", script.string8())};
 
 	JS::RootedObject nsAsObject{rq.cx, future.Get()};
 
@@ -414,7 +414,7 @@ Script::StructuredClone RunMapGeneration
 		if (!hasGenerator)
 		{
 			throw std::runtime_error{fmt::format(
-				"The map generation script {:?} didn't export a {:?}.", script.string8(),
+				"The map generation script {} didn't export a {}.", script.string8(),
 				GENERATOR_NAME)};
 		}
 	}
--- a/source/scriptinterface/FunctionWrapper.h
+++ b/source/scriptinterface/FunctionWrapper.h
@@ -515,7 +515,7 @@ public:
 				flags))
 			{
 				throw std::runtime_error{fmt::format(
-					"Failed defining function {:?} on the native scope.", classInfo->name)};
+					"Failed defining function {} on the native scope.", classInfo->name)};
 			}
 			JS::SetReservedSlot(functionObject, StatefullCallbackPrivateSlot::classInfo,
 				JS::PrivateValue(classInfo.release()));
